src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -85,77 +85,30 @@ class PythonSetup(Subsystem):
         ),
         advanced=True,
     )
-    requirement_constraints = FileOption(
```

Comment:
Moved to lower in the file, so that they show up later in help. We don't want any newcomers using this.